meta-iot2000-example/recipes-example/iot2000setup/files/expandfs.sh

```
@@ -1,31 +1,29 @@
 #!/bin/sh
 
-set -eu
```

Why did you remove this? At least `-u` has some value.

---

If something fails, for example sfdisk, the script should still disable itself that's why I removed set -e. -u does not help either... if some prog fails and we i.e. set ROOT_DEVICE=, then ROOT_DEVICE is no unbound variable anymore so that -u will not cause an exit of the shell. As the script is tested already, there are no misspellings either. So I decided to remove unneeded stuff.

---

Hm, if e.g. sfdisk fails, then there should be some notification to the user or something. As is, it fails silently, doesn't it?

---

I would say in our case, if sfdisk fails at that step, we have a problem with the whole system and it wont boot...

---

Yes, granted. Nonetheless, there should be a note left to the user that and what failed.